specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -172,6 +204,24 @@ The actual dockerfile can be found at [Dockerfile](/projects/frontend/cicd/Docke
 New versions are released by changing the version in [version.txt](/projects/frontend/cicd/version.txt)
 New releases are published under the image name [registry.hub.docker.com/versatiledatakit/vdk-cicd-base-gui](https://hub.docker.com/r/versatiledatakit/vdk-cicd-base-gui)
 
+#### Release
+
```

Comment:
While tests may more frequently run to verify dependents, only the modified artifacts get published.